integrations/cohere/pyproject.toml

```diff
@@ -47,6 +47,7 @@ dependencies = [
   "pytest",
   "pytest-rerunfailures",
   "haystack-pydoc-tools",
+  "jsonschema" # for tools
```

Comment:
```suggestion
```
We added the dependency to Haystack now: https://github.com/deepset-ai/haystack/blob/main/pyproject.toml#L60

Reply:
jsonschema is only part of Haystack's dependencies starting with the next release 2.10 but not before.